core/api/src/services/svix/index.ts

```diff
@@ -113,7 +113,7 @@ export const CallbackService = (config: SvixConfig) => {
   }
 
   // only work for hosted svix
-  const getWebsocketPortal = async (accountId: AccountId) => {
+  const getPortal = async (accountId: AccountId) => {
     const accountCallbackId = getAccountCallbackId(accountId)
```

Comment:
please include line 117-119 inside try block

Reply:
`getAccountCallbackId(accountId)` is just a string operation. it won't fail